plugins/modules/win_auto_logon.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Prasoon Karunan V (@prasoonkarunan)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_auto_logon
+short_description: Adds or Sets auto logon registry keys.
+description:
+  - Used to apply auto logon registry setting.
```

Comment:
Needs the `version_added` entry.

```suggestion
  - Used to apply auto logon registry setting.
version_added: 2.7.0
```

Reply:
Change added